react/Dots.js

```diff
@@ -0,0 +1,32 @@
+import { Component } from 'react'
+import PropTypes from 'prop-types'
+
+/**
+ * Dots container. Shows the nav dots of the carousel component;
+ */
+class Dots extends Component {
+  render() {
+    const { className, style, dots, color } = this.props
+
+    return (
+      <div className={className} >
+        <ul className="ma0 pa0" style={{ ... style, color: color || '#000' }}>
```

Comment:
Esse espaço entre `... style` não quebra?

Reply:
Tá funcionando tranquilo, mas eu removi o espaço mesmo assim just in case...